backend/src/main/java/org/eclipse/tractusx/puris/backend/common/edc/logic/service/EndpointDataReferenceService.java

```diff
@@ -51,7 +51,7 @@ public class EndpointDataReferenceService {
      * Please note that any data will only be stored for a period of 5
      * minutes. 
      * @param transferId
-     * @param authCode
+     * @param edr_Dto
```

Comment:
Please add small text to params. Sometimes feels redundant. But then rethink it and make it explicit.

Reply:
Done